tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_dsds_cell_data.py

```diff
@@ -17,26 +17,21 @@ def test_cell_data_for_two_sims(self):
               
         settings = Settings(self.marionette)
         settings.launch()
-
-        # Open cell data settings
         cell_and_data_settings = settings.open_cell_and_data_settings()
 
         # verify that a carrier is displayed
         self.assertTrue(len(cell_and_data_settings.carrier_name) > 0)
 
+        # enable cell data
+        self.assertFalse(cell_and_data_settings.is_data_toggle_checked)
         cell_data_prompt = cell_and_data_settings.enable_data()
 
         # deal with prompt that sometimes appears (on first setting)
```

Comment:
I don't like having an if in a test as it makes it not deterministic. What do you think @mwargers ?